sdk/tests/conformance2/glsl3/texture-offset-out-of-range.html

```diff
@@ -38,16 +38,27 @@
 <body>
 <div id="description"></div>
 <div id="console"></div>
+<script id="vshaderInvalidOffset" type="x-shader/x-vertex">#version 300 es
+in vec4 a_position;
+in vec2 a_in0;
+out vec2 v_texCoord;
+
+void main()
+{
+    gl_Position = a_position;
+    v_texCoord = a_in0;
+}
+</script>
 <script id="fshaderInvalidOffset" type="x-shader/x-fragment">#version 300 es
 precision mediump float;
 
+in vec2 v_texCoord;
 out vec4 my_FragColor;
 uniform sampler2D u_sampler;
-uniform vec2 u_texCoord;
```

Comment:
As I know, texture coordinates are seldom declared as uniform in shader functions.
On Windows with Intel GPU on Broadwell, WebGL 2 context will crash after the program finishes linking. The crash occurs in the driver (illegal memory access) and does not happen on a specific line of code in chromium.